docs/website/root/mithril/advanced/threat-model.md

```diff
@@ -309,25 +307,27 @@ Mithril genesis verification key is stored in [GitHub](https://github.com/input-
 
 #### Block production exhaustion
 
-- An incorrect mithril-signer could _starve_ the cardano-node of computing resources thus preventing it from producing and diffusing blocks in a timely manner
-- Compromising BP host would harms a BP's SPO economic viability
+- A malfunctioning or malicious `mithril-signer` could starve a `cardano-node` of resources, preventing it from producing and diffusing blocks in time.  
+- Compromising a block-producing node severely harms an SPO’s economic viability.
 
 ### Resource exhaustion on Mithril aggregator
 
-- DoS of a `mithril-aggregator`
-- Assets at risk:
+A DoS on the `mithril-aggregator`.
+- **Assets at risk**:
   - [Mithril signers registration](#mithril-signers-registration)
   - [Mithril signatures diffusion](#mithril-signatures-diffusion)
   - [Mithril certificates](#mithril-certificates)
   - [Mithril artifacts](#mithril-artifacts)
 
+
 ### Integrity of the Cardano block producer database
 
-- Data integrity of the Cardano block producer database compromised by action of the Mithril signer
-- Assets at risk:
+Data integrity of the Cardano block producer’s on-disk database could be compromised either by action of the Mithril signer or by an attacker with access to the signer.
+- **Assets at risk**:
   - [Block production](#block-production)
   - [Cardano Chain database](#cardano-chain-database)
-- Mitigation: give Mithril signer user read-only permissions to the database folder of the Cardano block producer
+
+- **Mitigation**: Assign the Mithril signer only read-only permissions to the Cardano block producer’s database folder.
 
 ## References
```

Comment:
The copy below needs edits to ensure 'Sentence case titles':

![image](https://github.com/user-attachments/assets/0039b997-12cd-4683-b82c-ea573540f50a)